impeller/renderer/backend/vulkan/formats_vk.h

```diff
@@ -138,6 +138,7 @@ constexpr vk::Format ToVKImageFormat(PixelFormat format) {
   switch (format) {
     case PixelFormat::kUnknown:
     case PixelFormat::kB10G10R10XR:
+    case PixelFormat::kB10G10R10A10XR:
```

Comment:
Also add reverse conversion to `ToPixelFormat` below.

Reply:
There's no reverse for this yet since it's mapped to `vk::Format::eUndefined`.  I don't think this format will ever be used with vulkan as far as I know.